components/Type.js

```diff
@@ -42,3 +48,62 @@ exports.H3 = Styled.Text`
     font-size: ${({ theme }) => theme.spacing(2.4)}px;
     font-weight: bold;
 `;
+
+const Link = ({
```

Comment:
Using this inline is, unfortunately, [still a bummer](https://stackoverflow.com/questions/66590167/vertically-align-pressable-inside-a-text-component).